packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTable.tsx

```diff
@@ -62,6 +70,73 @@ export const TransactionDateTime = ({
   )
 }
 
+export const CustomAddressTxExplorer = ({
```

Comment:
Reusable component that shows either 'Funds sent to' or 'Funds received from', followed by the explorer URL leading to a wallet address.

This is used both in the tx history row and in pending tx cards.

Reply:
<img width="413" alt="Screenshot 2023-07-19 at 12 58 20" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/47932951/137a6f4f-1a6a-417a-ba27-959a33c485fe">
<img width="233" alt="Screenshot 2023-07-19 at 12 58 38" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/47932951/a6b19737-1d44-47fa-b8c9-db06dd2055ea">